.github/workflows/build.yml

```diff
@@ -8,8 +8,12 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-18.04
-
+    runs-on: ${{ matrix.os }}
+    strategy:
+        matrix:
+            os: [ubuntu-latest, macOS-latest, windows-latest]
+            java: [ '11', '11.0.3', '15', '15.0.5']
```

Comment:
Why are versions specified twice, once as generic and once with a specific minor version?

This will 2x the build verification time, thus the wait time between push and "PR is mergeable".

Reply:
To check if the supported version at the time of configuration works and also the lastest java sub version

Reply:
The build verification time just increased from 9 to 12 min as most of the builds run in parallel.